doc/man3/EVP_KDF_CTX_set_hkdf_mode.pod

```diff
@@ -0,0 +1,149 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF_CTX_set_hkdf_mode,
+EVP_KDF_CTX_add1_hkdf_info
+- HMAC-based Extract-and-Expand KDF support functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/kdf.h>
+
+ int EVP_KDF_CTX_add1_hkdf_info(EVP_KDF_CTX *kctx, unsigned char *info,
+                                int infolen);
+
+ int EVP_KDF_CTX_set_hkdf_mode(EVP_KDF_CTX *kctx, int mode);
+
+=head1 DESCRIPTION
+
+The EVP_KDF_HKDF algorithm implements the HKDF key derivation function.
+HKDF follows the "extract-then-expand" paradigm, where the KDF logically
+consists of two modules. The first stage takes the input keying material
+and "extracts" from it a fixed-length pseudorandom key K. The second stage
+"expands" the key K into several additional pseudorandom keys (the output
+of the KDF).
+
+EVP_KDF_CTX_add1_hkdf_info() sets the info value to the first B<infolen> bytes
+of the buffer B<info>. If a value is already set, it is appended to the
+existing value.
+
+EVP_KDF_CTX_set_hkdf_mode() sets the mode for the HKDF operation. There are
+three modes that are currently defined:
+
+=over 4
+
+=item EVP_KDF_HKDEF_MODE_EXTRACT_AND_EXPAND
```

Comment:
This says `EVP_KDF_HKDEF_MODE_EXTRACT_AND_EXPAND` rather than `EVP_KDF_HKDF_MODE_EXTRACT_AND_EXPAND` (note extra 'E')

Reply:
Thanks!